pagopa/io-app-design-system
# Skip the `entering` transition on the first render of `ButtonSolid`

## Short description
This PR adds a skip of the `entering` transition on the first render of `ButtonSolid`

## List of changes proposed in this pull request
- Restore previous code to avoid `entering` transition

### Preview

The `entering` transition in the first render we'd like to remove (play at **0.25/0.5×**) 

https://github.com/user-attachments/assets/b394559b-555e-455c-a4ca-37f7495bac2f




## How to test
Go to the **Buttons** page in the example app and check if there's any difference between `main` and this branch
